contracts/feature-tests/composability/mandos/forw_raw_sync_echo.scen.json

```diff
@@ -1,121 +0,0 @@
-{
-    "gasSchedule": "v3",
-    "steps": [
-        {
-            "step": "setState",
-            "accounts": {
-                "address:a_user": {
-                    "nonce": "0",
-                    "balance": "0"
-                },
-                "sc:vault": {
-                    "nonce": "0",
-                    "balance": "0",
-                    "code": "file:../vault/output/vault.wasm"
-                },
-                "sc:forwarder": {
-                    "nonce": "0",
-                    "balance": "0",
-                    "code": "file:../forwarder-raw/output/forwarder-raw.wasm"
-                }
-            }
-        },
-        {
-            "step": "scCall",
-            "txId": "call_execute_on_dest_context",
```

Comment:
Please don't delete this file, it is still a valid test.

Reply:
restored